test/proto/BUILD

```diff
@@ -80,39 +79,36 @@ java_proto_library(
 scalapb_proto_library(
     name = "test_proto_java_conversions",
     visibility = ["//visibility:public"],
-    with_flat_package = True,
-    with_java = True,
     deps = [
         ":test2",
-        ":test_proto_java_lib",
+        # ":test_proto_java_lib",
         "//test/proto2:test",
     ],
 )
 
 scalapb_proto_library(
     name = "test_proto",
     visibility = ["//visibility:public"],
-    with_grpc = True,
     deps = [":test_service"],
 )
 
-scala_proto_srcjar(
-    name = "test1_proto_scala",
-    deps = ["//test/proto2:test"],
-    generator = "@io_bazel_rules_scala//src/scala/scripts:scalapb_generator")
+# scala_proto_srcjar(
```

Comment:
I guess we want to enable these before merging? somehow? I think passing the generator should not be an option, but building the test1 target.